Workbooks/ADXCluster/AtResource/ResourceInsights.workbook

```diff
@@ -1247,7 +1246,7 @@
                     "size": 3,
                     "noDataMessage": "No results found from the time range. Try selecting another time range.",
                     "timeContext": {
-                      "durationMs": 0
+                      "durationMs": 604800000
```

Comment:
@guregini  same question

Reply:
This will be always overwritten  by the time filter parameter (604800000 ms == 7 days)